test/bcs.js

```diff
@@ -91,9 +93,22 @@ describe.only('BCS', () => {
             .with.lengthOf(4)
             .that.includes('pay_gas', 'add_gas', 'collect_gas', 'refund');
     });
+
+    it('should decode RelayerDiscoveryV0 object successfully', async () => {
+        const relayerDiscoveryV0 = bcsStructs.relayerDiscovery.RelayerDiscovery.parse(fromHEX(hexData.RelayerDiscoveryV0));
+
+        expect(relayerDiscoveryV0.id).to.equal('5dcab278dc93438e0705fc32023808927e09a29b1ae52eef6cb33b9250d9b871');
+        expect(relayerDiscoveryV0.name).to.equal('0');
+        checkIdAndSize(relayerDiscoveryV0.value.configurations, '5339d11ffc9ae10e448b36b776533e1f08c646ad0441c7a0d410b1e0e5d28e58', '1');
+        expect(relayerDiscoveryV0.value.version_control.allowed_functions[0].contents)
+            .to.be.an('array')
+            .with.lengthOf(3)
+            .that.includes('register_transaction', 'remove_transaction', 'get_transaction');
+    });
 });
 
 // This function is used by getting the test data in bytes from the object id
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
why is this needed?

Reply:
this function is used for obtaining the bcs bytes when adding new decoding tests. i don't want to remove this function, but the linter failed because it's unused. so i added this line so that the linter will skip the check here.